bin/kong-dashboard.js

```diff
@@ -162,8 +162,8 @@ function start(argv) {
       terminal.error("This version of Kong dashboard doesn't support Kong v0.9 and lower.");
       process.exit(1);
     }
-    if (semver.gte(version, '0.13.0')) {
-      terminal.error("This version of Kong dashboard doesn't support Kong v0.13 and higher.");
+    if (semver.gte(version, '0.14.0')) {
+      terminal.error("This version of Kong dashboard doesn't support Kong v0.14 and higher.");
```

Comment:
The messaging can be improved.

Since this PR does not support Routes and Services, I would suggest adding some banner/info in the UI to tell the users that kong-dashboard does not support Routes and Services yet. How about adding it in the [footer](https://github.com/PGBI/kong-dashboard/blob/55f793b844c02274b2825db1b62668f376c8f10b/src/html/index.html#L54)?